src/main/webapp/createAd.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
This file is using camelcase for it's name, but item-page is using dashes between. Preferably choose one and try to stick with it

Reply:
Fixed for now. This is part of a bigger discussion because now it is inconsistent with other pages, so I'll talk to team.